### PR TITLE
Update README with clarified Windows instructions

### PR DESCRIPTION
**Pull request by odow**
Our poor Windows users are ill-served by the current installation instructions.

Someone has been emailing me complaining of segfaults.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,36 +21,40 @@ licensed under the [Apache 2.0 license](https://github.com/scipopt/scip/blob/mas
...
 
-Once installed, set the `SCIPOPTDIR` environment variable to point to the
-installation path, that is, depending on your operating system,
+Once the binaries are installed, set the `SCIPOPTDIR` environment variable to
+temporarily point to the installation path (that is, depending on your operating system,
 `$SCIPOPTDIR/lib/libscip.so`, `$SCIPOPTDIR/lib/libscip.dylib`, or
-`$SCIPOPTDIR/bin/scip.dll` must exist.
-
-Then, install `SCIP.jl` using `Pkg.add` and `Pkg.build`:
+`$SCIPOPTDIR/bin/scip.dll` must exist). Then, install `SCIP.jl` using `Pkg.add`
```
Comment (odow) [inline README.md:48]: This needs tweaking because they report to me `For Windows, there is no "scip.dll".`

Comment (odow) [inline README.md:48]:
```suggestion
`$SCIPOPTDIR/bin/libscip.dll` must exist). Then, install `SCIP.jl` using `Pkg.add`
```
At the very least, this is what `build.jl` currently expects.

Comment (odow) [reply README.md]: It was `libscip.dll`
image

**Comment by matbesancon** [comment]
Thanks! This has been a pain for a while already, without any success investigating what's going on with the JLL nor bandwidth to go the conda route like the others

**Pull request merged**